### ISSUE TITLE
Hls EXT-X-DISCONTINUITY-SEQUENCE

### ISSUE THREAD

**Issue by surajkumar-sk** [google/shaka-player#3353]
What does EXT-DISCONTINUITY-SEQUENCE tag do ? 
I know it's a pretty basic questions . But its been bugging me. I read the documents of HLS and saw how its being parsed in shaka-player , but that just increased my confusion. So can anyone explain about this tag in layman terms .
Thank you.

**Comment by theodab** [comment]
Each `EXT-X-DISCONTINUITY` has an associated "Discontinuity Sequence Number". This number is calculated based on how many `EXT-X-DISCONTINUITY` tags the parser has already seen, rather than being part of the tag. This number has to remain constant between manifest updates in a livestream, so that (for example) we can remember the associated timestamp offset of each discontinuity.
However, not every livestream holds on to old content forever. If the livestream is evicting old content, then old `EXT-X-DISCONTINUITY` tags might be evicted too, which could change the calculated discontinuity sequence number between manifest updates. Thus, we have the `EXT-X-DISCONTINUITY-SEQUENCE` tag, which lists how many `EXT-X-DISCONTINUITY` tags were dropped before the current start of the manifest.

**Comment by michellezhuogg** [comment]
@surajkumar-sk Let us know if you're facing a specific bug or issue! Otherwise, we'll start to provide mentorship in the summer. Thank you for your understanding!

**Comment by surajkumar-sk** [comment]
@theodab Thank you , now the link between them makes some sense to me. 
@michellezhuogg sorry for this , understanding the parser but still being stuck at it because of discontinuity tag made me reach out for help. It won't happen again. Thank you.
